### ISSUE TITLE
Accessibility: Language Attribute French

### ISSUE THREAD

**Issue by T-T-Nguyen** [StatCan/WellbeingCheckUAT#255]
@Jiandong-Guo The word “FRANCAAS“ needs to have a language attribute Lang= FR so that it is read properly this is a language/accessibility requirement.

**Comment by Jiandong-Guo** [comment]
It is not a HTML tag, It is just a label(string), there is no attribute with it

**Comment by T-T-Nguyen** [comment]
@Jiandong-Guo Further investigation will be neccessary as this is a language requirement. Although it the voice message you provided in 256 says English the rest of the content is in French. 
There must be a better way voice broadcast that is closer to the French language.  _It is unintelligible in French since it reads French words in English.  This cannot pass accessibility._

**Comment by Jiandong-Guo** [comment]
@T-T-Nguyen, we can change the button label from "Français" to "French" in english setting, the whole screen will be all english, the voice over will speak the label all in english,  
In French setting, we can change the button label from "English" to "Anglaise",  the whole screen will be all in French, the voice over will speak the label all in French.  Do you agree ?

**Comment by zhuelle** [comment]
@Jiandong-Guo and @T-T-Nguyen , I think What Jiandong's proposing is not working because if the user is in English setting, it should show 'Français' and if the user is in French setting, it should show 'English'. This is matching the common approach. I would augure that when reading the page out, it should read 'English' when it is in French and it should read 'Français' when it is in English. This is match what displayed. I don't think this is an error. We can discuss this in our meeting. 
![20210423_085053 (002)](https://user-images.githubusercontent.com/58314003/115875098-db448700-a412-11eb-80e4-99a9df43c8ff.jpg)

**Comment by zhuelle** [comment]
@Jiandong-Guo please change the lable back to 'Français' for English version and 'Englis' for French version.

**Comment by zhuelle** [comment]
@Jiandong-Guo could you please record voice back using your phone and attach it on the ticket when the page is in English and the ios is in English? In addition, could you please do the same thing for the COVID 19 app if possible? This way, the SMA can listen to compare. Thanks!

**Comment by Jiandong-Guo** [comment]
Compare between Covid19 and wellbeing check in French setting(Image and voice)
[3.zip](https://github.com/StatCan/WellbeingCheckUAT/files/6407064/3.zip)![3](https://user-images.githubusercontent.com/58485300/116732961-77453400-a9b9-11eb-8190-49a13127d536.jpg)
![4](https://user-images.githubusercontent.com/58485300/116732942-70b6bc80-a9b9-11eb-92f6-b31b8fbeef71.jpg)
[4.zip](https://github.com/StatCan/WellbeingCheckUAT/files/6407066/4.zip)

**Comment by zhuelle** [comment]
@T-T-Nguyen Jiandong has attached voice back for COVID-19 app and Well-being App in the ticket using his phone. Could you please verify if it is ok after the comparison?

**Comment by T-T-Nguyen** [comment]
@Jiandong-Guo Ticket will remain open until the new Accessibility assignees complete their testing and provide feedback.
